README.md

```diff
@@ -13,15 +13,22 @@ This is a framework for detecting memory issues in Dart and Flutter applications
 | [leak_tracker_flutter_testing](pkgs/leak_tracker_flutter_testing/) | Leak tracking helpers intended for usage in Flutter tests. | [![pub package](https://img.shields.io/pub/v/leak_tracker_flutter_testing.svg)](https://pub.dev/packages/leak_tracker_flutter_testing) |
 | [leak_tracker_testing](pkgs/leak_tracker_testing/) | Leak tracking helpers intended for usage in tests. | [![pub package](https://img.shields.io/pub/v/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing) |
 
-## How to enable logs
+## Roadmap
+
+[Work in progress](https://github.com/dart-lang/leak_tracker/labels/P1)
+[Upvote and comment, to help with prioritization](https://github.com/dart-lang/leak_tracker/labels/P2)
+
+## Contribution notes
+
```

Comment:
Perhaps also add, 'For general contributing information, see https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md.'